src/components/ListOfTipsAndComments.vue

```diff
@@ -1,42 +1,18 @@
 <template>
   <div>
-    <div class="profile__actions">
-      <div
-        class="activity-ribbon"
-      >
-        <FilterButton
-          :class="{ active: activity === 'channel' }"
-          @click="activity = 'channel'; activeTab = 'tips'"
-        >
-          <IconChannel />
-          <span class="vertical-align-mid">
-            {{ $t('components.ListOfTipsAndComments.Channel') }}
-          </span>
-        </FilterButton>
-        <FilterButton
-          :class="{ active: activity === 'activity' }"
-          @click="activity = 'activity'"
-        >
-          <IconActivity />
-          <span class="vertical-align-mid">
-            {{ $t('components.ListOfTipsAndComments.Activity') }}
-          </span>
-        </FilterButton>
-      </div>
-      <a
-        :class="{ active: activeTab === 'tips' }"
-        @click="setActiveTab('tips')"
-      >
-        {{ $t('tips') }}
-      </a>
-      <a
-        v-if="activity === 'activity'"
-        :class="{ active: activeTab === 'comments' }"
-        @click="setActiveTab('comments')"
-      >
-        {{ $t('comments') }}
-      </a>
-    </div>
+    <ActivityRibbon
+      :tabs="ribbonTabs"
+      :activity="activity"
+      :set="(setActivity) => activity = setActivity"
```

Comment:
```suggestion
      v-model="activity"
```
🤷‍♀️

Reply:
this didn't seem to work, don't know why

Reply:
figured it out